Lib/fontmake/font_project.py

```diff
@@ -644,6 +644,18 @@ def run_from_designspace(
             logger.info('Applying instance data from designspace')
             ufos.extend(apply_instance_data(designspace_path,
                                             include_filenames=filenames))
+            logger.info("Expanding features to instance UFOs")
+            master_source = next(
+                (s for s in designspace.sources if s.copyFeatures), None
+            )
+            if not master_source:
+                logger.info("... actually, no source is marked as the feature master")
```

Comment:
Let’s avoid too verbose logging messages. Either demote to DEBUG or remove completely